### ISSUE TITLE
SearchEverywhere broken in 2022.3: NoSuchMethodError

### ISSUE THREAD

**Issue by soportetierranube** [wt-io-it/odoo-pycharm-plugin#99]
Plugin Version: 0.6.7
IntelliJ Version: PyCharm 2022.3
IntelliJ Build: PC-223.7571.203

<details><summary>java.lang.NoSuchMethodError: 'void com.intellij.ide.actions.searcheverywhere.FoundItemDescriptor.<init>(java.lang.Object, int, double)'</summary>

```
java.lang.NoSuchMethodError: 'void com.intellij.ide.actions.searcheverywhere.FoundItemDescriptor.<init>(java.lang.Object, int, double)'
	at at.wtioit.intellij.plugins.odoo.search.OdooFoundItemDescriptor.<init>(OdooFoundItemDescriptor.java:10)
	at at.wtioit.intellij.plugins.odoo.search.OdooFoundItemDescriptor.weighted(OdooFoundItemDescriptor.java:23)
	at at.wtioit.intellij.plugins.odoo.search.OdooSEContributor.lambda$fetchWeightedElements$0(OdooSEContributor.java:71)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:847)
	at at.wtioit.intellij.plugins.odoo.search.OdooSEContributor.fetchWeightedElements(OdooSEContributor.java:62)
	at com.intellij.ide.actions.searcheverywhere.MixedResultsSearcher$ContributorSearchTask.run(MixedResultsSearcher.java:182)
	...
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:246)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	...
	at java.base/java.lang.Thread.run(Thread.java:833)
```
</details>

<details><summary>induced.txt</summary>

```
Following exceptions happened soon after this one, most probably they are induced.


2022-12-12T10:18:50.883
java.lang.NoSuchMethodError: 'void com.intellij.ide.actions.searcheverywhere.FoundItemDescriptor.<init>(java.lang.Object, int, double)'
	at at.wtioit.intellij.plugins.odoo.search.OdooFoundItemDescriptor.<init>(OdooFoundItemDescriptor.java:10)
	at at.wtioit.intellij.plugins.odoo.search.OdooFoundItemDescriptor.weighted(OdooFoundItemDescriptor.java:23)
	at at.wtioit.intellij.plugins.odoo.search.OdooSEContributor.lambda$fetchWeightedElements$2(OdooSEContributor.java:121)
	at at.wtioit.intellij.plugins.odoo.WithinProject.run(WithinProject.java:25)
	at at.wtioit.intellij.plugins.odoo.search.OdooSEContributor.lambda$fetchWeightedElements$3(OdooSEContributor.java:101)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:847)
	at at.wtioit.intellij.plugins.odoo.search.OdooSEContributor.fetchWeightedElements(OdooSEContributor.java:100)
	at com.intellij.ide.actions.searcheverywhere.MixedResultsSearcher$ContributorSearchTask.run(MixedResultsSearcher.java:182)
	...
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:246)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	...
	at java.base/java.lang.Thread.run(Thread.java:833)

```
</details>

**Comment by ap-wtioit** [comment]
Documentation for removed Constructor: https://github.com/JetBrains/intellij-sdk-docs/commit/c725cff7d117cbc71c4f0ee075aae0f915d81473

**Comment by ap-wtioit** [comment]
Fix updated in version 0.6.8 of the plugin (pending approval by jetbrains) for those eager to test the new version can also be found here:
https://github.com/wt-io-it/odoo-pycharm-plugin/tree/v0.6.8

**Comment by bigbear3001** [comment]
0.6.8 is now available in the marketplace
